polling-consumer/polling-consumer.bal

```diff
@@ -0,0 +1,41 @@
+import ballerina/http;
+import ballerina/lang.runtime;
+
+enum Status {
```

Comment:
Here we can reduced the number of enum values. Because we are just mocking the paypal api.